docs/guides/qiskit-code-assistant-local.mdx

```diff
@@ -0,0 +1,204 @@
+---
+title: Use Qiskit Code Assistant in local mode
+description: Learn how to deploy and use the Qiskit Code Assistant model locally.
+---
+
+{/* cspell:ignore ollama, gguf */}
+
+# Use Qiskit Code Assistant in local mode
+
+Learn how to install, configure, and use the Qiskit Code Assistant model on your local machine.
+
+<Admonition type="note" title="Notes">
+    - Qiskit Code Assistant is in preview release status and is subject to change.
+    - If you have feedback or want to contact the developer team, use the [Qiskit Slack Workspace channel](https://qiskit.enterprise.slack.com/archives/C07LYA6PL83) or the related public GitHub repositories.
+</Admonition>
+
+## Download the Qiskit Code Assistant model
+
+The Qiskit Code Assistant model is available in  <DefinitionTooltip definition="GGUF is a binary format that is designed for fast loading and saving of models, and for ease of reading.">GGUF file format</DefinitionTooltip> and can be downloaded from the Hugging Face Hub in one of two ways.
+
+<details>
+
+<summary>Download from the Hugging Face website</summary>
+
+Follow these steps to download the Qiskit Code Assistant GGUF model from the Hugging Face website:
+
+1. Navigate to the `granite-8b-qiskit` model page: https://huggingface.co/qiskit
+1. Select the `granite-8b-qiskit` GGUF model version
+1. Go to the Files and Versions tab and download the GGUF model
+
+</details>
+
+
+<details>
+
+<summary>Download using the Hugging Face CLI</summary>
+
+To download the `granite-8b-qiskit` GGUF model using the Hugging Face CLI follow these steps:
+
+1. Install the Hugging Face CLI: https://huggingface.co/docs/huggingface_hub/main/en/guides/cli
+1. Login to your Hugging Face account
+
+    ```
+    huggingface-cli login
+    ```
+
+1. Download the `granite-8b-qiskit` GGUF model
+
+    ```
+    huggingface-cli download <HF REPO NAME> <GGUF PATH> --local-dir <LOCAL PATH>
+    ```
+
+</details>
+
+
+## Get the Qiskit Code Assistant model up and running
+
+There are multiple ways to deploy and interact with the downloaded `granite-8b-qiskit` GGUF model. These instructions outline how to get up and running on your local machine using [Ollama](https://ollama.com).
+
+- [Using the Ollama application](#using-the-ollama-application)
+- [Using the `llama-cpp-python` package](#using-the-llama-cpp-python-package)
```

Comment:
```suggestion
```suggestion
There are multiple ways to deploy and interact with the downloaded `granite-8b-qiskit` GGUF model, using [Ollama](https://ollama.com): either with the [Ollama application](#using-the-ollama-application), or with the [`llama-cpp-python` package](#using-the-llama-cpp-python-package).
```
```

Reply:
hi @abbycross thanks for the updates/review. what i wanted to say here is that there are multiple ways to run to run the model. Ollama is just one application (there are others like [Jan AI](https://jan.ai/docs), etc.). 
basically, as long as an application supports the GGUF model format that this model can be used. we just happen to choose Ollama for this guide

Reply:
Got it, I understand - so the multiple ways don't just involve the two Ollama options. We can reword to something like:

```suggestion
There are multiple ways to deploy and interact with the downloaded `granite-8b-qiskit` GGUF model. This guide demonstrates using [Ollama](https://ollama.com) as follows: either with the [Ollama application](#using-the-ollama-application), or with the [`llama-cpp-python` package](#using-the-llama-cpp-python-package).
```
Would something like this make more sense?

Reply:
yes, thank you! i appreciate it.